extractor/src/main.py

```diff
@@ -56,13 +55,15 @@ def create_app():
 # Override default JSONEncoder
 app.json_encoder = MyJSONEncoder
 
+redis_client = redis.Redis(host=REDIS_MAPPINGS_HOST, port=REDIS_MAPPINGS_PORT, db=REDIS_MAPPINGS_DB)
```

Comment:
Maybe we should add a comment to point out that we have a redis client for each thread?

Reply:
I'm changing this in order to have a single one